core/services/cable_guy/api/manager.py

```diff
@@ -274,9 +271,7 @@ def remove_ip(self, interface_name: str, ip_address: str) -> None:
             interface_index = self._get_interface_index(interface_name)
             self.ipr.addr("del", index=interface_index, address=ip_address, prefixlen=24)
         except Exception as error:
-            error_msg = f"Cannot delete IP '{ip_address}' from interface {interface_name}. {error}"
-            logger.error(error_msg)
-            raise ValueError(error_msg) from error
+            raise RuntimeError(f"Cannot delete IP '{ip_address}' from interface {interface_name}.") from error
```

Comment:
Is it possible to visualize such errors also via the logger interface ?

Reply:
Yes, it will be there, because of the stacked_trace_message.